RELEASE.md

```diff
@@ -12,6 +12,20 @@
 1. Copy/paste CHANGELOG into the [draft release](https://github.com/fastly/cli/releases).
 1. Publish draft release.
 
+## Creation of npm modules
+
+Each release of the Fastly CLI will trigger the creation of a new version of the `@fastly/cli` npm module, as well as multiple per-platform modules (e.g. `@fastly/cli-darwin-arm64`). The root module (`@fastly/cli`) depends on the platform-specific modules as `optionalDependencies` so that npm will only install the compatible platform-specific package.
+
+To see an example of the module layout, run...
+
+$ npm install @fastly/cli-darwin-arm64 --verbose
```

Comment:
Can we wrap this example shell line (and the following) in a code-block please.

Otherwise great 👍🏻